ProjectLighthouse.Servers.Website/Pages/Partials/CommentsPartial.cshtml

```diff
@@ -16,91 +16,102 @@
     {
         <p>There are no comments.</p>
     }
+    else if (Model.ProfileUser.IsBanned)
+    {
+        <p>Comments are disabled because the user is banned.</p>
+    }
     else if (!Model.CommentsEnabled)
     {
         <b>
             <i>Comments are disabled.</i>
         </b>
     }
+    else if (Model.Comments.Count >= 50)
+    {
+        <p>There are more than 50 comments. Displaying the newest ones.</p>
+    }
```

Comment:
From my knowledge, the latest 50 comments are fetched which would set the comments count to 50, no?

Reply:
You're right. The model comment count can never go over 50, so your options are to introduce a CommentCount variable in UserPage and SlotPage or to scrap the idea and leave it how it is until pagination gets implemented.

Reply:
> or to scrap the idea and leave it how it is until pagination gets implemented.
I guess this would be okay since it's really just a visual nitpick more than anything.